CHANGELOG.md

```diff
@@ -3,6 +3,8 @@
 ## Master (Unreleased)
 
 * Fix an error for `RSpec/Capybara/SpecificFinders` with no parentheses. ([@ydah][])
+* Exclude methods that raise `ArgumentError` with `RSpec/Capybara/VisibilityMatcher` where `:visible` is an invalid option. ([@ydah][])
```

Comment:
Perhaps it's better to list which elements were excluded.

Reply:
That looks better. I have modified it to list it.

Reply:
I think now you have to entries for the same commit

Reply:
Oops, looks like the second commit contained an unnecessary entry. Fixed it.